src/wing/CCPACSWingCell.cpp

```diff
@@ -407,246 +444,325 @@ TopoDS_Shape CCPACSWingCell::GetSkinGeometry(TiglCoordinateSystem cs) const
     }
 }
 
-void CCPACSWingCell::BuildSkinGeometry(GeometryCache& cache) const
+TopoDS_Shape CCPACSWingCell::CutSpanwise(GeometryCache& cache,
```

Comment:
Is it possible to extract a more generic function that cuts a shell using an iso-curve from a face?

Reply:
I don't think it can be done easily, because if the border is not an iso-curve, we must cut, possibly using ribs or spars. But I suppose the code blocks associated to the trimming can be reused to avoid code duplication.

Reply:
Yes, I thought more of a generic function that trims a face using an iso curve. This gets in particular interesting, if the face has non trivial boundaries.